### ISSUE TITLE
Accuracy is not changing| RNN example

### ISSUE THREAD

**Issue by MrGrimod** [aymericdamien/TensorFlow-Examples#161]
Hello,
I am using [this:](https://github.com/aymericdamien/TensorFlow-Examples/blob/master/examples/3_NeuralNetworks/recurrent_network.py) example.
When I run it, loss is decreasing but accuracy is not changing.
The weights are not being updated as well, I checked that by using:
`variables_names =[v.name for v in tf.trainable_variables()]
            values = session.run(variables_names)
            for k,v in zip(variables_names, values):
                print(k, v)`
